src/app/shared/notification/notification.service.ts

```diff
@@ -71,24 +77,43 @@ export class NotificationService {
   }
 
   // toast message pop up, by default, shown success message for 2 seconds.
-  async presentToast(message, success = true, duration?) {
-    let color = 'success';
-    if (!success) {
-      color = 'danger';
-    }
-    return this.customToast({
+  async presentToast(message, success = false, duration?, customOptions?: ToastOptions) {
+    let toastOptions: ToastOptions = {
       message: message,
-      duration: duration || 2000,
+      duration: duration || 200000,
```

Comment:
@sasangachathumal  why 3mins++ ? 😅

Reply:
@trtshen i'm still workin on this that added for testing. without making duration long can't inspect and check styles of toast. and this is not ready for review or testing. i'm still working on it.

Reply:
okok, i thought it's final, it's alright then, thanks @sasangachathumal